docs/console-api/models/index.js

```diff
@@ -3,6 +3,8 @@ export default {
   autotune: require("./autotune.json"),
   dynamic_config: require("./dynamic_config.json"),
   dynamic_config_update: require("./dynamic_config_update.json"),
```

Comment:
these were all using_underscores but now you did `camelCase` for `experimentMetric` - stick to one

Reply:
Will fix when I push the overrides docs, the user shouldn't be able to see this